bin/cli.js

```diff
@@ -149,18 +150,32 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
+const shouldValidate = (options, validationList) => {
+  return options._.every(command => {
+    const mainCommand = command;
+    if (validationList[mainCommand]) {
+      if (validationList[mainCommand].skip) {
+        return false;
+      }
+      const subCommands = validationList[mainCommand].subCommands || {};
```

Comment:
We'll need to account for arbitrarily-nested subcommands, which I think means recursion is the best option here

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1310/commits/312431ebc20d2fc2129c0f5d655fde96b4aff3e1